gp-nested-forms/gpnf-hide-expired-nested-form-field.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Gravity Perks // GP Nested Forms // Hide Nested Form Field if the Nested Form has expired.
+ * https://gravitywiz.com/documentation/gravity-forms-nested-forms/
+ *
+ * Instruction Video: https://www.loom.com/share/798f30d73d254864aa6a65e7d16d390b
+ */
+add_filter( 'gform_pre_render', 'check_nested_form_expiration' );
+function check_nested_form_expiration( $form ) {
+	// Update '435' to the targeted Parent Form's ID.
+	if ( rgar( $form, 'id' ) != 435 || ! is_callable( 'gp_nested_forms' ) ) {
```

Comment:
Thoughts on making this accept an array? Might also be worth making it so you can pass some sort of value that makes it global.